src/System.Security.Cryptography.Cng/src/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs

```diff
@@ -319,6 +361,15 @@ internal bool ReleaseNativeWithNCryptFreeObject()
     /// </summary>
     public sealed class SafeNCryptKeyHandle : SafeNCryptHandle
     {
+        public SafeNCryptKeyHandle()
+        {
+        }
+
+        public SafeNCryptKeyHandle(IntPtr handle, SafeHandle parentHandle)
```

Comment:
It needs to be callable by the X509Certificates library.  It's new API that is a consequence of the X509Certificates feature.